lightly/utils/io.py

```diff
@@ -63,21 +64,28 @@ def check_embeddings(path: str):
     # first col is `filenames`
     if header[0] != 'filenames':
         raise RuntimeError(f'Embeddings csv file must start with `filenames` '
-                           f'column but had {header[0]} instead.')
-    
+                        f'column but had {header[0]} instead.')
+
+    # last column is `labels`
```

Comment:
I think it has to. The way we defined the format is here: https://docs.lightly.ai/getting_started/command_line_tool.html#id1

it might be that some parts are more relaxing when using the embeddings (mundig, workers, etc. ) but the original format is this one. I would also keep it that way in the check here so we at least have some consistency.